libnavui-dropin/build.gradle

```diff
@@ -43,7 +43,7 @@ android {
 }
 
 dependencies {
-    api project(":libnavui-base")
+    implementation project(":libnavui-app")
```

Comment:
That's because we don't want this dependency to be exported outside. `libnavui-app` is an internal module used only by `libnavui-dropin` and `libnavui-androidauto`.

If we use the **api** configuration, the `libnavui-app`  will be transitively exposed to consumers of either library, and as such will appear on the compile classpath of consumers. Using the **implementation** configuration will, on the other hand, not be exposed to consumers, and therefore not leak into the consumers compile classpath.

Reply:
For similar reasons [above](https://github.com/mapbox/mapbox-navigation-android/pull/5826#discussion_r877303536) why do we need it to be internal?